src/js/videojs-dash.js

```diff
@@ -81,66 +54,14 @@ var dashjs = require('dashjs');
     this.mediaPlayer_.attachView(this.el_);
 
     // Dash.js autoplays by default
-    if (!options.autoplay) {
-      this.mediaPlayer_.setAutoPlay(false);
-    }
-
-    // Fetches and parses the manifest - WARNING the callback is non-standard "error-last" style
-    this.mediaPlayer_.retrieveManifest(manifestSource, videojs.bind(this, this.initializeDashJS));
-  }
-
-  Html5DashJS.prototype.initializeDashJS = function (manifest, err) {
-    var manifestProtectionData = {};
-
-    if (err) {
-      Html5DashJS.showErrors(this.elParent_);
-      this.tech_.triggerReady();
-      this.dispose();
-      return;
-    }
+    this.mediaPlayer_.setAutoPlay(false);
```

Comment:
okay it may be good to add a comment about it being handled in video.js just so it does not confuse anyone